core/charm/strategies.go

```diff
@@ -49,6 +50,13 @@ func AlwaysChecksum(string) bool {
 	return true
```

Comment:
Same comment as below for the function name. It might be a bit cleaner to call it something like `AlwaysMatchChecksum`

Reply:
Agreed